src/controllers/Lane.js

```diff
@@ -198,7 +198,7 @@ class Lane extends Component {
           getChildPayload={index => this.props.getCardDetails(id, index)}>
           {cardList}
         </Container>
-        {editable && !addCardMode && <components.AddCardLink onClick={this.showEditableCard} t={t} />}
+        {editable && !addCardMode && <components.AddCardLink onClick={this.showEditableCard} t={t} laneId={id} />}
```

Comment:
@harshaktg Hi!

Good idea. Could you make an example of usage in StoryBook?

Reply:
Sure. I'll make and example and raise another MR for the same